kolibri/core/auth/constants/collection_kinds.py

```diff
@@ -12,6 +12,6 @@
 # the ordering of kinds in the following tuple corresponds to their level in the hierarchy tree
 choices = (
     (FACILITY, _("Facility")),
-    (CLASSROOM, _("Classroom")),
+    (CLASSROOM, _("Class")),
```

Comment:
Would it risk regression too much if we were to change `CLASSROOM` to `CLASS` throughout for consistency?

Reply:
We used `Classroom`, `CLASSROOM` etc to avoid accidental collisions with the Python reserved word `class`, and would be nice not to change constant variable names if at all possible (although their human readable values like here are fine).

Reply:
Also, although these are wrapped in a translation wrapper, I am not sure these particular strings are ever exposed to the user.